python/pipeline/ProjectorColor.m

```diff
@@ -0,0 +1,9 @@
+%{
```

Comment:
I think the name of the class should agree with the name of the file. But also, I don't think this table should be here. It is already under pipeline/experiment.py

Reply:
still unresolved (we either need to put this in experiment or in pipeline, I think it makes more sense in experiment as all other projector stuff is there)

Reply:
deleted. Idk why it was still there